### PR TITLE
Typography & Colors

### PR DESCRIPTION
**Pull request by thenorthkun**
Added the following components / sub-components -

Typography

- Headings
- Font weights
- Font size

Colors

- Default Palette

**Typography preview :** https://fezco-ui-git-typography-colors-thenorthkun.vercel.app/components/typography/typography.html
**Colors preview :** https://fezco-ui-git-typography-colors-thenorthkun.vercel.app/components/colors/colors.html

### CODE REVIEW

File: components/colors/colors.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    padding: 0.5rem;
+    font-weight: 550;
+    font-size: 1rem;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+}
+
+.primary{
+    background-color: #03fc96;
```
Comment (j24m) [inline components/colors/colors.css]: color variables?

File: components/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    flex-direction: column;
+    width: 60%;
+    margin-bottom: 3rem;
+    gap: 10px;
+    border-radius: 4px;
+    min-height: 100px;
+    padding: 1rem;
+    background-color: #f5f7f9;
```
Comment (j24m) [inline components/typography/typography.css:12]: color variable?

**Review by j24m — APPROVED** [review]

**Comment by poornima-alugubelly** [comment]
I really love that green dot u have after each component heading : )

File: components/colors/colors.css (hunk 1 above)
Comment (poornima-alugubelly) [reply components/colors/colors.css]: make variable primary-bg then resuse

**Review by poornima-alugubelly — APPROVED** [review]

**Pull request merged**